Sources/TuistServer/Session/ServerSessionController.swift

```diff
@@ -18,6 +18,23 @@ public enum DeviceCodeType {
     }
 }
 
+public enum ServerSessionControllerError: Equatable, FatalError {
```

Comment:
```suggestion
enum ServerSessionControllerError: Equatable, FatalError {
```

I believe this error can stay internal.

Reply:
TuistKitTests/Services/WhoamiServiceTest references it now, so it has to be public I believe? Unless I should test that another way without referencing the error directly.

Reply:
We could add `@testable` annotation to `@testable import TuistServer` instead, but this is also fine. Let's keep it 👍